### ISSUE TITLE
iOS13 上会导致状态栏的点击失效，直接的影响就是导致全局单击状态栏（tableView/collectionView）返回顶部失效

### ISSUE THREAD

**Issue by WXXXXH** [devxoul/Toaster#183]
v:2.2.0
iOS13

**Comment by WXXXXH** [comment]
ToastWindow.shared.windowLevel = 10000  iOS 13 上虽然没有点击事件但是会阻止

**Comment by WXXXXH** [comment]
解决方案：继承了Toast，重写main方法，add的时候显示ToastWin，dissmiss的时候 show

**Comment by cozzin** [comment]
@WXXXXH Hi, do you mean we should only show `ToastWindow` when you have toast?

**Comment by WXXXXH** [comment]
> @WXXXXH Hi, do you mean we should only show `ToastWindow` when you have toast?

yes,you can test in iOS 13,if your project hava collectionView/tableView,，and Toast
.show(),then you will find click stateBar not scroll to top，

**Comment by cozzin** [comment]
I'm struggling with same issue https://github.com/devxoul/Toaster/issues/180.
Do you know what causes this issue?

**Comment by WXXXXH** [comment]
> I'm struggling with same issue #180.
> Do you know what causes this issue?

iOS 13 Bug，可能不允许多个UIwindow ，当系统检测到多个UIwindow的时候 事件会传递给另一个等级更高的？猜测。。。目前iOS13 字符串也有一些问题

**Comment by cozzin** [comment]
Ok, I will test with multiple windows and other windowLevels.

**Comment by cozzin** [comment]
This issue is resolved in iOS 13.2.
@WXXXXH Could you check in iOS 13.2?
